androgee/androgee.py

```diff
@@ -6,23 +6,26 @@
 
 
 logging.basicConfig(level=logging.WARNING)
-bot = commands.Bot(command_prefix=os.environ['DISCORD_PREFIX'])
+bot = commands.Bot(command_prefix=os.environ["DISCORD_PREFIX"])
+
 
 @bot.event
 async def on_ready():
-    print('We is logged in as {0.user}'.format(bot))
+    print("We is logged in as {0.user}".format(bot))
 
 
 @bot.command(name="spray")
-async def spray(ctx):
+async def spray(ctx, member: discord.Member):
```

Comment:
Do we _always_ want to include a member? Is there a time where we just want to call `-spray` without mentioning a member?